department-of-veterans-affairs/caseflow
# Make establish claim page

Added a new establish claim page written in React. Fixes: #416 

- Updated the EstablishClaim container to reflect the design
- Added new components to use on the page

Added tests.
- Used the new testing framework for React tests

Fixed some issues around the new React setup.
- Mainly added jsdom to allow us to run full featured React tests

**Test Plan**
- Verify the page confirms to the design
<img width="927" alt="screen shot 2016-11-21 at 10 09 18 pm" src="https://cloud.githubusercontent.com/assets/3885236/20509513/3160152a-b037-11e6-940a-ab400aee611c.png">
- Run new and old tests

**To Do**
- [x] Make sure jslint runs
- [x] Make tests for components

@joofsh @sharonwarner
